pkg/net/libp2p/libp2p.go

```diff
@@ -209,13 +203,6 @@ func WithRoutingTableRefreshPeriod(period time.Duration) ConnectOption {
 	}
 }
 
-// WithBootstrapMinPeerThreshold set a minimal peer threshold for bootstrap process.
-func WithBootstrapMinPeerThreshold(threshold int) ConnectOption {
-	return func(options *ConnectOptions) {
-		options.BootstrapMinPeerThreshold = threshold
-	}
-}
```

Comment:
Heads up, we still use this function in `keep-ecdsa` in `start.go`:
```
libp2p.WithBootstrapMinPeerThreshold(bootstrapMinPeerThreshold),
```

Reply:
Yep, I know. We can just remove it when we will update `keep-core` dependency.